drivers/clock_control/clock_stm32_ll_u5.c

```diff
@@ -899,6 +899,18 @@ int stm32_clock_control_init(const struct device *dev)
 		return -ENOTSUP;
 	}
 
+#ifdef CONFIG_PM
+	/* Disable unused clocks that are enabled. These will not be enabled, unless
+	the MCU uses them for PM wakeup purposes. */
+	if (!IS_ENABLED(STM32_MSIS_ENABLED) && READ_BIT(RCC->CR, RCC_CR_MSISON)) {
```

Comment:
Should replace `READ_BIT(RCC->CR, RCC_CR_MSISON)`
with `(READ_BIT(RCC->CR, RCC_CR_MSISON) != 0U)`

Ditto at line below: `(READ_BIT(RCC->CR, RCC_CR_HSION) != 0U)`